### DISCUSSION TITLE
wallpapers not working

### DISCUSSION THREAD

**Discussion by stickyautism** [Almamu/linux-wallpaperengine#213] [Q&A]
**Wallpaper Engine Background(s)**
https://steamcommunity.com/sharedfiles/filedetails/?id=2797732338
https://steamcommunity.com/sharedfiles/filedetails/?id=1677412765
https://steamcommunity.com/sharedfiles/filedetails/?id=2915841260
https://steamcommunity.com/sharedfiles/filedetails/?id=2246687467
https://steamcommunity.com/sharedfiles/filedetails/?id=2310614396
https://steamcommunity.com/sharedfiles/filedetails/?id=1882919717
https://steamcommunity.com/sharedfiles/filedetails/?id=1094833813
https://steamcommunity.com/sharedfiles/filedetails/?id=2276320172
https://steamcommunity.com/sharedfiles/filedetails/?id=2310614396
**Console output**
Cannot find directory for steam app wallpaper_engine: assets
No "scene.pkg" file found at "/home/slimyiscool/Downloads/wallpaper-enginefiles/2310614396/scene.pkg". Defaulting to normal folder storage
No "gifscene.pkg" file found at "/home/slimyiscool/Downloads/wallpaper-enginefiles/2310614396/gifscene.pkg". Defaulting to normal folder storage
terminate called after throwing an instance of 'WallpaperEngine::Assets::CAssetLoadException'
  what():  Cannot find file : Cannot find directory
zsh: IOT instruction (core dumped)  ./linux-wallpaperengine

**Desktop:**
 
- OS: Pop!_OS
 - Desktop Environment: GNOME

**Comment by Almamu** [comment]
Based on the console output you're missing the assets folder from WallpaperEngine, please check the README.MD: https://github.com/Almamu/linux-wallpaperengine?tab=readme-ov-file#52-extracting-the-assets

**Comment by asmcint** [comment]
I have the same issue, same console output, and I have followed the directions and copied the assets folder to the same directory where the binary lives. It doesn't work. This entire process of setting this up at all has been immensely frustrating in the first place (hint: "build it yourself lmao" is never a valid philosophy for software installation, especially when the intended function of the software is naturally going to attract Windows expats), and to then have it all  "working" only for the basic function of the software to fail is utterly maddening.

EDIT: Tried a different wallpaper, got a different result. Preview window opened for a split second, closed, and I got this terminal output:

[1125/220530.617433:WARNING:resource_util.cc(94)] Please customize CefSettings.root_cache_path for your application. Use of the default value may lead to unintended process singleton behavior.
Detected "scene.pkg" file at "/home/user/.steam/steam/steamapps/workshop/content/431960/2065528681/scene.pkg". Adding to list of searchable paths
No "gifscene.pkg" file found at "/home/user/.steam/steam/steamapps/workshop/content/431960/2065528681/gifscene.pkg". Defaulting to normal folder storage
WARNING: Shader patches directory cannot be found, this might make some backgrounds not work properly
Resorting to default value as type "options" is unknown
Resorting to default value as type "options" is unknown
Resorting to default value as type "audioprocessingoptions" is unknown
Resorting to default value as type "imageblending" is unknown
Resorting to default value as type "options" is unknown
Shutting down CEF
X Error of failed request:  BadWindow (invalid Window parameter)
  Major opcode of failed request:  15 (X_QueryTree)
  Resource id in failed request:  0x3e578e20
  Serial number of failed request:  22
  Current serial number in output stream:  22

**Reply by ghost** [reply]
I think your command output is other issues, I recomment you to create issues
